src/test-reader/index.js

```diff
@@ -20,7 +20,7 @@ module.exports = class TestReader extends EventEmitter {
     }
 
     async read(options = {}) {
-        const { paths, browsers, ignore, sets, grep } = options;
+        const { paths, browsers, ignore, sets, grep, runnableOpts } = options;
```

Comment:
Add new option `runnableOpts` which has only one nested option `saveLocations` in order to collect all `line` and column` values for each suite and test.